data/wanderer/wanderers middle.txt

```diff
@@ -20,6 +20,319 @@ mission "Wanderers: Pre-Colony"
+	cargo "prefabricated building materials" 500
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 1: offered"
+	
+	on stopover
+		dialog "The dockworkers are talking excitedly as they load the building materials. When you ask, the dock foreman tells you they are exited to be helping establish the first settlement in the new worlds."
+
+	on visit
+		dialog "You've arrived at <planet> but the escort carrying the <cargo> hasn't landed. You realize you need to depart and wait for it."
+
+	on complete
+		dialog "A team of workers unload the <cargo> while the foreman checks the manifest before handing you a credit chip for <payment>."
+		payment 200000 2
+
+mission "Wanderers: Spera Anatrusk Outfitter 2c"
+	landing
+	name "Heavy machinery to <planet>"
+	description "Bring <cargo> from <stopovers> to <destination> to build a new outfitter. Payment is <payment>."
+	blocked "The building supervisor at <destination> has [ordered, requisitioned] more equipment than you can carry. You will need <capacity> to transport it."
+	source "Vara K'chrai"
+	destination "Spera Anatrusk"
+	stopover "Varu Tek'kai"
+	cargo "heavy machinery" 300
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 1: offered"
+
+	on stopover
+		dialog "You notice that each crate of machinery has been signed by the dock team that loaded it."
+
+	on visit
+		dialog "You've arrived at <planet> but the escort carrying the <cargo> hasn't landed. You realize you need to depart and wait for it."
+
+	on complete
+		dialog "The construction supervisor drops by as the workers are unloading. He gives you a credit chip for <payment> and asks you to meet him in the tent that serves as the spaceport when all the deliveries are done."
+		payment 250000 3
+
+
+mission "Wanderers: Spera Anatrusk Outfitter 3"
+	landing
+	name "Warehouse equipment to <planet>"
+	description "Deliver warehouse equipment and staff for the outfitter on <destination>."
+	source "Spera Anatrusk"
+	destination "Vara K'chrai"
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 2a: done"
+		has "Wanderers: Spera Anatrusk Outfitter 2b: done"
+		has "Wanderers: Spera Anatrusk Outfitter 2c: done"
+
+	on offer
+		conversation 
+			`This time you land on a concrete pad and you see the familiar hexagonal layout of a Wanderer spaceport. There are tents instead of buildings but you can see stakes outlining the shapes to come. You pass a group of Wanderer captains who appear to be holding a swap meet looking for spare ammunition and equipment so they can avoid the trip back across the Eye.`
+			`	The construction supervisor bobs his head enthusiastically when he sees you and eventually remembers to switch to Hai. "Thank you, Captian <last>," he says. "We would be waiting weeks to get started building without your help. We now need to get the warehouse equipment and [personnel, staff] in to organize the outfitter and [technicians, fixers] do do the [maintenance, repairs]. Would you be able to do this for us as well?`
```

Comment:
```suggestion
			`	The construction supervisor bobs his head enthusiastically when he sees you and eventually remembers to switch to Hai. "Thank you, Captain <last>," he says. "We would be waiting weeks to get started building without your help. We now need to get the warehouse equipment and [personnel, staff] in to organize the outfitter and [technicians, fixers] do the [maintenance, repairs]. Would you be able to do this for us as well?`
```

Reply:
Just a thought: That extra "do" should probably actually be a "to".